.eslintrc.json

```diff
@@ -41,6 +41,7 @@
 		"parser": "@typescript-eslint/parser",
 		"plugins": [ "@typescript-eslint" ],
 		"rules": {
+			"no-undef": "off",
```

Comment:
ESLint gives false warnings about some DOM types not being defined, such as `RequestInit`. As per ESLint [themselves](https://eslint.org/docs/latest/rules/no-undef#handled_by_typescript), this rule should not be enabled for typescript files, since the TypeScript compiler already checks for those types:

> It is safe to disable this rule when using TypeScript because TypeScript's compiler enforces this check. 

Here it how it looks:

![image](https://github.com/user-attachments/assets/f3018df0-2de8-441b-967a-b90c14f252de)